cbrain_task/recon_all/bourreau/recon_all.rb

```diff
@@ -139,7 +147,23 @@ def cluster_commands #:nodoc:
 
     recon_all_command = "recon-all#{lbl_ext} #{with_qcache} #{with_mprage} #{with_3T_data} #{with_cw256} #{with_hippocampal} -sd . #{subjid_info} #{step} #{with_notal_check} #{lbl_options}"
 
+    # Copy license
+    cp_license = <<-CP_LICENSE
```

Comment:
Usually using the `-HERE` syntax is to allow the token to be indented, yet it isn't indented later on.